<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>axis_settings</class>
 <widget class="QDialog" name="axis_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Axis</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/new/icon_resource/RS.ico</normaloff>:/new/icon_resource/RS.ico</iconset>
  </property>
  <widget class="QPushButton" name="ok_push_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>207</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="xAxis_group_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>230</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>X</string>
   </property>
   <widget class="QLabel" name="xMax_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QLabel" name="xMin_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xMin_line_edit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>18</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="xMax_line_edit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="yAxis_group_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>230</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Y</string>
   </property>
   <widget class="QLabel" name="yMax_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QLabel" name="yMin_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yMin_line_edit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>18</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="yMax_line_edit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>xMin_line_edit</tabstop>
  <tabstop>xMax_line_edit</tabstop>
  <tabstop>yMin_line_edit</tabstop>
  <tabstop>yMax_line_edit</tabstop>
  <tabstop>ok_push_button</tabstop>
  <tabstop>xMin_line_edit_2</tabstop>
  <tabstop>xMax_line_edit_2</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
